### PR TITLE
Handle Chat Message Notifications (#316)

### PR DESCRIPTION
**Pull request by andrewDoing**
- Shuffle appropriate ChatListItem to the top.
- Add the ChatListItem and call load() if not in the list.
- Update the preview message.
- Change the class to "highlighted".
- Call onMessageReceived() prop from PlexChat.
- Validate
     | 'chatMessageReceived'

### CODE REVIEW

File: samples/react-chat/src/App.tsx
Comment (seekdavidlee) [inline samples/react-chat/src/App.tsx]: actual remove the word plex since this is customer info - the final PR should not have customer specific info

File: packages/mgt-chat/src/components/ChatList/ChatList.tsx
Comment (plasne) [inline packages/mgt-chat/src/components/ChatList/ChatList.tsx:124]: Is this old? I thought @seekdavidlee removed onChatListEvent()

**Review by plasne — CHANGES_REQUESTED** [review]
This seems to have a bunch of stuff from other PRs waiting to merge. I want to hold off approving until I see those merge and all that out of here. I think because some of this looks out-of-date.

File: packages/mgt-chat/src/components/ChatList/ChatList.tsx
Comment (andrewDoing) [reply packages/mgt-chat/src/components/ChatList/ChatList.tsx:124]: I added it back to handle this case, I'm guessing there's another way to accomplish this?

**Review by TechnicallyWilliams — APPROVED** [review]

**Review by plasne — APPROVED** [review]

**Pull request merged**
